lemmy_apub/src/fetcher.rs

```diff
@@ -258,8 +258,12 @@ pub(crate) async fn get_or_fetch_and_upsert_user(
         recursion_counter,
       )
       .await?;
-      // TODO: do we need to set this? would need a separate db call
-      //uf.last_refreshed_at = Some(naive_now());
+
+      let user_id = user.id;
+      blocking(context.pool(), move |conn| {
+        User_::mark_as_updated(conn, user_id)
```

Comment:
Its not just the updated column, but the last_refreshed_at column too. 

Before the code was:

```
      let mut uf = UserForm::from_apub(
        &person?,
        context,
        Some(apub_id.to_owned()),
        recursion_counter,
      )
      .await?;
      uf.last_refreshed_at = Some(naive_now());
      let user = blocking(context.pool(), move |conn| User_::update(conn, u.id, &uf)).await??;
```

Reply:
Sorry, fixed.